batch.go

```diff
@@ -243,14 +233,30 @@ func newLabelPool(faker *fake.Faker, cardinalities map[string]int) LabelPool {
 		"format": []string{"apache_common", "apache_combined", "apache_error", "rfc3164", "rfc5424", "json", "logfmt"}, // needs to match the available flog formats
 		"os":     []string{"darwin", "linux", "windows"},
 	}
+
+	num := 10
+	var nss []string
 	if n, ok := cardinalities["namespace"]; ok {
```

Comment:
this part is confusing to me, not sure why you use these default values here again. they are defined in
https://github.com/co-brick/xk6-loki/blob/main/loki.go#L160
so if len(labels) == 0 they will be initialized anyway